plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2837,15 +2870,15 @@ private void testStatsBasedRepartitionData(boolean ctas)
         });
     }
 
-    private void testRepartitionData(Session session, String sourceRelation, boolean ctas, String partitioning, int expectedFiles)
+    protected void testRepartitionData(Session session, String sourceRelation, boolean ctas, String partitioning, int expectedFiles)
     {
         String tableName = "repartition" +
                 "_" + sourceRelation.replaceAll("[^a-zA-Z0-9]", "") +
                 (ctas ? "ctas" : "insert") +
                 "_" + partitioning.replaceAll("[^a-zA-Z0-9]", "") +
                 "_" + randomTableSuffix();
 
-        long rowCount = (long) computeScalar(session, "SELECT count(*) FROM " + sourceRelation);
+        long rowCount = computeActual("SELECT * FROM " + sourceRelation).getRowCount();
```

Comment:
It will not work then wiht ORDER BY and LIMIT clause needed in
io.trino.plugin.iceberg.BaseIcebergMinioConnectorTest
 - testRepartitionDataOnCtasPartitionComplex
 - testRepartitionDataOnInsertPartitionComplex
 - testRepartitionDataOnCtasPartitionWithSameColumnMultipleTimes
 - testRepartitionDataOnInsertPartitionWithSameColumnMultipleTimes

Reply:
`"SELECT * FROIM (" + sourceQuery ")"` ( + https://github.com/trinodb/trino/pull/10894/files#r810120979 )